doc/source/parameters/dem/insertion_info.rst

```diff
@@ -133,8 +148,21 @@ The ``file`` insertion method inserts particles in a similar way to the ``list``
 
 Each line is associated with a particle and its properties. The ``fem_force`` and ``fem_torque`` properties are only used in the CFD-DEM solver, but must be specified in all cases. The main advantage of using the ``file`` method over the ``list`` method is that the number of inserted particles is not limited to the maximum number of characters on a single line of parameter files. To generate an insertion file, particle positions and properties can be generated manually or with any script. An other option is to use the python code ``extract-particles-properties-from-vtu.py`` in ``lethe/contrib/preprocessing/`` directory. This code extracts particle properties from the last vtu file from a given simulation.
 
+* ``insertion frequency`` defines the frequency of the insertion of particles based on the list in the file(s)
+
+* ``list of input files`` defines the list of files to be used for the insertion. The default value is ``particles.input``.
```

Comment:
You should add something regarding the sequential order of files. Like, for the first insertion, the first file will be used, second insertion the second file will be used, etc. 

If the end of the list is reached, next insertion will use the first file, etc.

Reply:
Yeah good addition!

Reply:
Actually it is said in the intro of the method.

Reply:
Fogot about it then